server.js

```diff
@@ -1,23 +1,43 @@
-/* eslint global-require: 0 */
+/* eslint comma-dangle: 0, global-require: 0, import/first: 0, no-console: 0 */
 
 const production = (process.env.NODE_ENV === 'production')
+
+import 'babel-polyfill'
```

Comment:
I'll see what I can do. We can't conditionally use `import` like we could with `require` so might still have to have new relic be first thing but I can consolidate the statement